templates/search_page.html

```diff
@@ -136,10 +137,19 @@ <h1 style="color: #24c1e0; text-align: center; background-color: whitesmoke">
 
                     <div class="flex-fill">
                         <input class="form-control border-0" type="search" id="location" placeholder="Search"
-                            aria-label="Search" name="location" value="Los Angeles, CA, USA">
+                            aria-label="Search" name="location" value="Los Angeles, CA, USA" onchange="fetchLocation(this)"> 
                     </div>
 
-                    <div class="fixedwidthicon">
+                    <script>
+                        function fetchLocation() {
```

Comment:
Should we invoke this function when user hit search?

Reply:
That is correct, this is invoked when the user hits the `enter` for search with the location